src/js/tracks/text-track-settings.js

```diff
@@ -58,7 +158,7 @@ function captionOptionsMenuTemplate(uniqueId, dialogLabelId, dialogDescriptionId
               </select>
             </span>
           </fieldset>
-          <fieldset class="window-color vjs-tracksetting">
+          <fieldset class="vjs-window-color vjs-tracksetting">
```

Comment:
will this still be backwards compatible (as it used to be called `window-color`)?